bin/create-scripts/block.js

```diff
@@ -31,6 +32,27 @@ const config = {
 	},
 };
 
+const funcName = 'gutenberg_register_block_' + blockName.replace( '-', '_' );
+const blockRegistration = `<?php
+add_action( 'enqueue_block_editor_assets', '${ funcName }' );
+function ${ funcName }() {
+	wp_enqueue_style(
+		'${ blockName }',
+		plugins_url( '/blocks/blocks-${ blockName }.css', __FILE__ ),
+	);
+
+	wp_enqueue_script(
+		'${ blockName }',
+		plugins_url( '/blocks/blocks-${ blockName }.js', __FILE__ ),
+	);
```

Comment:
I really foresee us making multiple blocks with `build block` and I'm not sure this PHP is going to cut it for loading all of them. 

I'm also curious about printing this to `stdout` vs. in a file because I suspect that in most cases the PHP won't be applicable due to plugin-specific circumstances that require special loading code (see a8c-blocks)

I _do_ want to spit out the list of dependencies at some point through code analysis though I'm not sure that's necessary now.

Reply:
Thanks for the feedback, Dennis!

> I really foresee us making multiple blocks with build block and I'm not sure this PHP is going to cut it for loading all of them.

Well, it's fair to say that the JS build process we have right now might not be sufficient as well, and would need to be improved and expanded. This is only a first version, of course, and it will be iterated on. FWIW, there will surely be differences in all environments that we'll be loading those blocks in, so we might need to end up with different PHP for different environments. Another thing that might vary is that dynamic blocks need to be registered on the server side. Also, I heard all block registration might be moved to the server side soon, so we'll have to look out for that as well. That being said, this is yet another piece that we will iterate on, and this is only the first version.

> I'm also curious about printing this to stdout vs. in a file because I suspect that in most cases the PHP won't be applicable due to plugin-specific circumstances that require special loading code (see a8c-blocks)

We could do that, but it could also get ugly when we have a bunch of dependencies at the end. We could add that as an argument to the build script.

> I do want to spit out the list of dependencies at some point through code analysis though I'm not sure that's necessary now.

As early as it is now, we still don't have a good way to manage dependencies, let's keep iterating and add to what we have.

What do you think?